assets/scss/global.scss

```diff
@@ -1,24 +1,55 @@
-$maxWidth : 1440px;
-$contentWidth : 1256px;
+$maxWidth: 1440px;
+$contentWidth: 1366px;
 
 body {
   background: #ffffff;
 }
 
-section{
+section {
   display: flex;
   flex-direction: column;
   align-items: center;
   width: 100%;
-  .content{
+  .content {
     width: 100%;
     max-width: $contentWidth;
     padding: spacingXY(10, 18);
-    @include media("<medium") {
-      padding: spacingXY(6,4);
+    @include media('<medium') {
+      padding: spacingXY(6, 4);
     }
-    &.max-width{
+    &.max-width {
       max-width: $maxWidth;
     }
   }
+  &.max-width {
+    max-width: $maxWidth;
+  }
+}
+
+button.p-button-sm {
```

Comment:
From looking at this, it seems you are using a base html button? and not a PrimeVue p-button?

Reply:
no biggie.. when this all get's merged... I'll take a peek and adjust

Reply:
oh yeah, I thought you meant just a normal button - duh!